generators/client-2/templates/src/main/webapp/app/account/password/_password.controller.ts

```diff
@@ -1,32 +1,42 @@
-(function() {
-    'use strict';
+import { Component, Inject, OnInit } from "@angular/core";
+import { TranslatePipe } from '../../shared/translate.pipe';
 
-    angular
-        .module('<%=angularAppName%>.account')
-        .controller('PasswordController', PasswordController);
+@Component({
+    selector: 'password',
+    templateUrl: 'app/account/password/password.html',
+    pipes: [TranslatePipe]
```

Comment:
this is not required, as with rc6 onwards this will be declared in the ngModule declaration.I have already done that so you can remove this